landoui/treestatus.py

```diff
@@ -211,14 +211,24 @@ def treestatus_tree(tree: str):
     """Display the log of statuses for an individual tree."""
     api = TreestatusAPI.from_environment()
 
-    logs_response = api.request("GET", f"trees/{tree}/logs")
-    logs = logs_response.get("result")
+    try:
+        logs_response = api.request("GET", f"trees/{tree}/logs")
+
+        # We got a response, if this is empty is it an error.
+        logs = logs_response.get("result")
+        detail = "empty response from Lando"
+    except LandoAPIError as exc:
+        if not exc.detail:
+            raise exc
+
+        # We got some exception back from LandoAPI..
+        detail = exc.detail
+        logs = []
```

Comment:
I don't think that settings `logs` to an empty list is necessary, there are a number of 4xx and 5xx errors that can be returned by Lando API that do not mean there are no logs.